src/main/java/com/smartbear/readyapi/client/execution/CodegenBasedTestServerApi.java

```diff
@@ -296,8 +300,41 @@ public void setBasePath(String basePath) {
     }
 
     @Override
-    public ProjectResultReport postProject(ProjectExecutionRequest executionRequest, boolean async, HttpBasicAuth auth) throws ApiException {
+    public ProjectResultReport postSwagger(File swaggerFile, TestServerRequestExecutor.SwaggerFormat swaggerFormat,
```

Comment:
yeah - I think that makes sense - it will keep the different pieces of functionality separated and avoided overly cluttered interfaces... !?